week-11-front-end-frameworks-and-mobile/day-02/drill/math-fundamentals.md

```diff
@@ -0,0 +1,350 @@
+  <tr>
+    <td><strong>Complex Numbers</strong></td>
+    <td>A combination of real and imaginary numbers of the form a + bi where a and b are real numbers and i is the square root of -1 (see imaginary number). While real numbers can be represented as points on a line, complex numbers can only be located on a plane. See Types of Numbers.</td>
+  </tr>
+  <tr>
+    <td><strong>Composite Number</strong></td>
+    <td>Any integer which is not a prime number, i.e., evenly divisible by numbers other than 1 and itself.</td>
+  </tr>
+  <tr>
+    <td><strong>Congruent</strong></td>
+    <td>Alike in all relevant respects.</td>
+  </tr>
+  <tr>
+    <td><strong>Constant</strong></td>
+    <td>A quality of a measurement that never changes in magnitude.</td>
+  </tr>
+  <tr>
+    <td><strong>Coordinate</strong></td>
+    <td>A set of numbers that locates the position of a point usually represented by (x, y) values.</td>
+  </tr>
+  <tr>
+    <td><strong>Cosine Law</strong></td>
+    <td>For any triangle, the side lengths a, b, c and corresponding opposite angles A, B, C are related as follows: a2 = b2 + c2 - 2bc cosA, etc. The law of cosines is useful to determine the unknown data of a triangle if two sides and an angle are known. See Wikipedia: Cosine Law.</td>
+  </tr>
+  <tr>
+    <td><strong>Cube Root</strong></td>
+    <td>The factor of a number that, when it is cubed (i.e., x^3) gives that number.</td>
+  </tr>
+  <tr>
+    <td><strong>Decimal</strong></td>
+    <td>A fraction having a power of ten as denominator, such as 0.34 = 34/100 (102) or 0.344 = 344/1000 (103). In the continent, a comma is used as the decimal point (between the unit figure and the numerator).</td>
+  </tr>
+  <tr>
+    <td><strong>Denominator</strong></td>
+    <td>The bottom number in a fraction.</td>
+  </tr>
+  <tr>
+    <td><strong>Diameter</strong></td>
+    <td>A straight line that passes from side to side thorough the centre of a circle.</td>
+  </tr>
+  <tr>
+    <td><strong>Digit</strong></td>
+    <td>In the decimal system, the numbers 0 through 9.</td>
+  </tr>
+  <tr>
+    <td><strong>Division</strong></td>
+    <td>The operation of ascertaining how many times one number, the divisor, is contained in another, the dividend. The result is the quotient, and any number left over is called the remainder. The dividend and divisor are also called the numerator and denominator, respectively.</td>
+  </tr>
+  <tr>
+    <td><strong>Even Number</strong></td>
+    <td>A natural number that is divisible by two.</td>
+  </tr>
+  <tr>
+    <td><strong>Exponent (Power, Index)</strong></td>
+    <td>
+      A number denoted by a small numeral placed above and to the right of a numerical quantity, which indicates the number of times that quantity is multiplied by itself. In the case of X^n, it is said that X is raised to the power of n. When aand b are non-zero real numbers and p and q are integers, the following rules of power apply:<br><br>
+      ap x aq = ap + q<br><br>
```

Comment:
We should use `x^n` to denote exponents, rather than `xn` (since that looks like multiplication). I wasn't sure what all the exponents were in these rules, so could you go through and format them with `^` as needed? There are also a few other places throughout these notes where exponents are present in the examples - I think I caught most of them, but can you double check if any are missing the `^`?